lib/python/file_classifier.py

```diff
@@ -90,3 +130,99 @@ def _make_path(cls, dir_list):
         for i in range(len(dir_list)):
             dir_list[i] = str(dir_list[i])
         return os.path.join(*dir_list)
+
+
+
+class MooringFileClassifier(FileClassifier):
+
+    PROJECT = 'IMOS'
+
+    @classmethod
+    def _get_data_category(cls, input_file):
+        """Determine the category a file belongs to (Temperature,
+        CTD_timeseires, Biogeochem_profile, etc..)
+
+        """
+
+        feature_type = cls._get_nc_att(input_file, 'featureType')
+        if feature_type == 'profile':
+            return 'Biogeochem_profiles'
+
+        var_names = set(cls._get_variable_names(input_file))
+        salinity = set(['PSAL', 'CNDC'])
```

Comment:
Those variables can probably become a hash. And then you will not need a lot of `if`s down below, but something to iterate on the hash.